File: app/src/main/java/com/leonyr/mvvmframe/GitHubApi.java
package com.leonyr.mvvmframe;

import com.leonyr.mvvmframe.model.Momodiy;

import io.reactivex.Observable;
import retrofit2.http.GET;

/**
 * ==============================================================
 * Description:
 * <p>
 * Created by 01385127 on 2019.04.29
 * (C) Copyright sf_Express Corporation 2014 All Rights Reserved.
 * ==============================================================
 */
public interface GitHubApi {

    @GET("/users/momodiy")
    Observable<Momodiy> gitMomodiy();

}

File: app/src/main/java/com/leonyr/mvvmframe/GitHubApi.kt
package com.leonyr.mvvmframe

import com.google.gson.JsonObject
import retrofit2.Response
import retrofit2.http.Body
import retrofit2.http.POST

/**
 * ==============================================================
 * Description:
 *
 *
 * Created by 01385127 on 2019.04.29
 * (C) Copyright sf_Express Corporation 2014 All Rights Reserved.
 * ==============================================================
 */
interface GitHubApi {

    @POST("api/login")
    suspend fun login(@Body params: JsonObject): Response<UserModel>

}
